mats/5_3.ms

```diff
@@ -2346,7 +2346,7 @@
     (error? (expt 0 -1))
     (error? (expt 0 +1i))
     (eqv? (expt 2+2i 4) -64)
-    (eqv? (expt 10.0 -20) 1e-20)
+    (fl~= (expt 10.0 -20) 1e-20)
```

Comment:
I'm not comfortable removing this precise test from the operating systems that implement the pow function correctly. What do you think of using a case statement with (machine-type) to express the expected result and the broken result on the operating systems that don't get it quite right?

Reply:
Casing on the machine type seems strange to me. If it's a bug that `expt` loses precision, then it seems like running the test should report that.

Calling it a bug does make sense, but then I'm inclined to say that the problem is outside the scope of this PR and just revert the test changes.

Reply:
I agree on both counts. It does look like a bug outside the scope of this pull request that we don't want to mask.